src/java.base/linux/native/libsimdsort/avx2-32bit-qsort.hpp

```diff
@@ -0,0 +1,367 @@
+/*
+ * Copyright (c) 2021, 2023, Intel Corporation. All rights reserved.
+ * Copyright (c) 2021 Serge Sans Paille. All rights reserved.
```

Comment:
Is this person an OpenJDK Contributor?

Reply:
Not listed here: https://oca.opensource.oracle.com/?ojr=contributors

Reply:
Yes, Vamsi is part of Intel Java team. He also has the author status (https://openjdk.org/census#sparasa).

Reply:
@sviswa7 I am asking about the copyright holder "Serge Sans Paille".

Reply:
> @sviswa7 I am asking about the copyright holder "Serge Sans Paille".

Hi David (@dholmes-ora),

Sorry for the late reply as I was on vacation last week.

The SIMD sort for Java/OpenJDK is based on Intel's [x86-simd-sort](https://github.com/intel/x86-simd-sort) library. Serge Sans Paille (@serge-sans-paille) was one of the contributors to the x86-simd-sort. That is the reason his name is listed in the copyright notice. 

Thanks,
Vamsi

Reply:
I take it he is not an Intel employee, in which case he has to be an OpenJDK contributor himself for code for which he holds a copyright to be contributed to OpenJDK. @robilad please correct me if I am wrong here.

Reply:
hey o/ No problem on my side to either let go my copyright or fill the contributor agreement (where is it?)

Reply:
Thanks Serge! And thank you for sending in an OCA for processing, which has now been done.